build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
```

Comment:
Top level project doesn't need any plugins applied.

Reply:
actually I tried removing them from here and things broke.  I guess I'll fool around with it some more.